README.md

```diff
@@ -1,56 +1,42 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Tour Booking App (backend)
 
-> Description the project.
+> This is an app in which you can reserve a tour, create a tour, delete a tour and show all your reserved tours. This app was created by separating the Fron-End and the Back-End into two repositories.
 
+## API documentation
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
+- https://booking-tour-app.herokuapp.com/api-docs/index.html
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
+## Built With
 
-### Install
+- Ruby on Rails
+- Postgres
 
-### Usage
+## Setup
```

Comment:
- [ ]  Please, kindly add instructions about how to run your tests.  

Be careful.  Upon running just `rspec` this happens:

![image](https://user-images.githubusercontent.com/57421823/199580537-66f8bc6f-39e8-40d5-929e-bdaada0da4b0.png)

Suggestion:  Instead of instructing your users to just run `rspec`, instruct them to run `rake rswag`:

![image](https://user-images.githubusercontent.com/57421823/199580799-955ad422-c3f5-49b7-880d-dec929df5023.png)

- [ ]  Also, as seen on the last screenshot, kindly delete the tests that were autogenerated and not being used.

That way, your apps integrity is preserved, you clear the warnings, and your users will never get a failed test 💪